app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "com.example.firemap"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Any reason behind this?

Reply:
Ah, I should have left a comment here as well. Down in line 56 of Conversion.kt, it explains why. Basically, because we want the user to choose a destination directory, not a file, the specified action should be different. And the action to choose a directory is only supported in SDK 21+.

Although, now that I think about it, I probably could have checked to see if the file they chose was a directory or not afterwards, but this is a little cleaner.